apps/web/src/lib/shortAddress.ts

```diff
@@ -0,0 +1,4 @@
+export const shortAddress = (addr: string): string =>
```

Comment:
a unit test would be good to add for this helper

Reply:
Removed this helper